clients/python-wrapper/lakefs/object_io.py

```diff
@@ -35,28 +41,49 @@ def __init__(self, repository: str, reference: str, path: str,
         self._pos = 0
 
     @property
-    def pos(self):
+    def pos(self) -> int:
```

Comment:
```suggestion
    def tell(self) -> int:
```
and as a method, not property..